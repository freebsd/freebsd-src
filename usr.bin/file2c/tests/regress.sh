
echo 1..9

REGRESSION_START($1)

REGRESSION_TEST(`1', `head -c 13 ${SRCDIR}/regress.in | file2c')
REGRESSION_TEST(`2', `head -c 26 ${SRCDIR}/regress.in | file2c PREFIX')
REGRESSION_TEST(`3', `head -c 39 ${SRCDIR}/regress.in | file2c PREFIX SUFFIX')
REGRESSION_TEST(`4', `head -c 52 ${SRCDIR}/regress.in | file2c -x')
REGRESSION_TEST(`5', `head -c 65 ${SRCDIR}/regress.in | file2c -n -1')

REGRESSION_TEST(`6', `head -c  7 ${SRCDIR}/regress.in | file2c -n 1 P S')
REGRESSION_TEST(`7', `head -c 14 ${SRCDIR}/regress.in | file2c -n 2 -x "P S"')
REGRESSION_TEST(`8', `head -c 21 ${SRCDIR}/regress.in | file2c -n 16 P -x S')

REGRESSION_TEST(`9', `file2c "const char data[] = {" ", 0};" <${SRCDIR}/regress.in')

REGRESSION_END()
