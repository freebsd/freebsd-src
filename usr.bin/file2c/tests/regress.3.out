PREFIX
175,212,59,239,227,234,192,22,186,113,52,204,195,81,240,223,173,115,109,
102,175,183,24,185,173,137,107,116,77,142,217,172,196,45,62,228,41,174,
134
SUFFIX
